### PR TITLE
Imperia after launch bugfixes

### PR DESCRIPTION
**Pull request by HiddeLekanne**

### CODE REVIEW

File: gui/budget_panel.gui [hunk 1]
```diff
@@ -1376,7 +1376,6 @@ types budget
 								fontsize = 45
 								autoresize = yes
 								margin = { 10 10 }
-								align = center
```
Comment (Danarca) [inline gui/budget_panel.gui:1379]: Why are you removing the `align` segments? Are they invalidated by some other bits in there?

Comment (HiddeLekanne) [reply gui/budget_panel.gui:1379]: In this case there is another align right above it. In other cases the error log reports that they are not used.

Comment (Danarca) [reply gui/budget_panel.gui:1379]: Have you checked it out ingame? Does it look fine without the aligns?

Comment (HiddeLekanne) [reply gui/budget_panel.gui:1379]: Just to reiterate this one had two align = center

Comment (Danarca) [reply gui/budget_panel.gui:1379]: Yea, I'm more so interested in the ones without any aligns at all now :p

Comment (HiddeLekanne) [reply gui/budget_panel.gui:1379]: I only briefly checked the vienna congress

**Review by Danarca — APPROVED** [review]
You're right, seems to be no noticeable differences from the aligns. Errors have also dropped like a rock, I had 133 after country selection!

**Pull request merged**